docs/phoenix.csv

```diff
@@ -844,9 +844,10 @@ MIN(NAME)
 "
 
 "Functions (Aggregate)","SUM","
-SUM( { numericTerm } )
+SUM( [ DISTINCT ] { numericTerm } )
```

Comment:
For now parser changes are fine. But the CALCITE work may lead to some changes here. But for now fine.